core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -213,6 +213,11 @@ oppia.factory('oppiaPlayerService', [
                 _explorationId));
           }
           loadedExploration.then(function(data) {
+            // Update exploration's states field.
+            for (var stateName in data.exploration.states) {
```

Comment:
@seanlip There are other instances that do not have access to this data. Precisely why I thought I'll do it here since Player services is the only service that receives the exploration for the learner. The learner view is the only place where we ever need the state_classifier_mapping.